pkgs/by-name/fo/foxglove-studio/package.nix

```
@@ -0,0 +1,130 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  dpkg,
+  glibc,
+  gcc-unwrapped,
+  autoPatchelfHook,
+  makeDesktopItem,
+  makeWrapper,
+  copyDesktopItems,
+  alsa-lib,
+  at-spi2-atk,
+  at-spi2-core,
+  atk,
+  cairo,
+  cups,
+  dbus,
+  expat,
+  gtk3,
+  libGL,
+  libX11,
+  libXcomposite,
+  libXdamage,
+  libXext,
+  libXfixes,
+  libXrandr,
+  libdrm,
+  libxcb,
+  libxkbcommon,
+  mesa,
+  nspr,
+  nss,
+  pango,
+  udev,
+  x11basic,
+}:
+stdenv.mkDerivation rec {
+  name = "foxglove-studio-${version}";
```

```suggestion
  pname = "foxglove-studio";
```
`name` already gets generated as `pname-version`, try not to define it explicitly